PopPUNK/lineage_clustering.py

```diff
@@ -0,0 +1,680 @@
+#!/usr/bin/env python
+# vim: set fileencoding=<utf-8> :
+# Copyright 2018-2020 John Lees and Nick Croucher
+
+# universal
+import os
+import sys
+import re
+# additional
+import numpy as np
+from scipy.stats import rankdata
+from collections import defaultdict
+import pickle
+from multiprocessing import Pool, Lock, Manager, RawArray, shared_memory, managers
```

Comment:
Add an import error warning here, and note that python >=3.8 is required

Reply:
Added in da5b6df826fe1cafe622094f82cb8d3677354661.